rust-cargo.el

```diff
@@ -19,6 +19,13 @@
   :type 'boolean
   :group 'rust-mode)
 
+(defcustom rust-locate-project-in-workspace t
+  "Whether to use `--workspace` with `cargo locate-project`. If t,
+ rust-mode will run commands for the entire workspace. If nil,
+ rust will search for the Cargo.toml in the local crated"
+  :type 'boolean
+  :group 'rust-mode)
+
```

Comment:
How about we change this to `rust-cargo-locate-default-arguments` ? It's default value could be `(list "--workspace")`. For your specific use case, you can pass it as `nil`.

This will give you the ability to pass any arbitrary arguments to `locate-project`.

Reply:
Done